Update orbit interlock HLIOC

anacso17 opened the pull request:
- Update LLRF configurations to consider new Input 1 (bit 5) as orbit interlock input
- Add new IA-06RaBPM:TI-EVR to the list of timing redundancy devices

fernandohds564 commented on siriuspy/siriuspy/orbintlk/main.py:
```diff
@@ -1249,8 +1249,8 @@ def _check_configs(self):
                 value = _updt_bit(value, 2*i, 0)
-                fim_orbit = dev.fast_interlock_monitor_orbit
-                fim_manual = dev.fast_interlock_monitor_manual
+                fim_orbit = dev['FIMLLRF1-Sts']
```
just out of curiosity, why are you abandoning the use of the python property here @anacso17 ?

anacso17 replied:
Hi @fernandohds564 , this was a first step for testing, in a next commit I would replace the hard-coded strings by attributes depending on the LLRF, this would make it possible to have different inputs for each LLRF (as we needed the last months)

fernandohds564 replied:
thanks for the explanation!

fernandohds564 commented on siriuspy/siriuspy/orbintlk/main.py:
```diff
@@ -1510,10 +1512,10 @@ def _handle_reliability_failure(self, is_failure=True):
         _time.sleep(1)
         for llrf in self._llrfs:
-            llrf.interlock_manual = 0
+            llrf['IntlkManual-Sel'] = 0
```
same here.

anacso17 replied:
In fact here we could use the property

ericonr reviewed (approved):
Changes suggested by fernando haven't been pushed yet. Besides that, LGTM

ericonr commented on siriuspy/siriuspy/orbintlk/main.py:
```diff
@@ -1510,10 +1523,10 @@ def _handle_reliability_failure(self, is_failure=True):
         self._update_log('FATAL:sending soft interlock to LLRF.')
         # sending interlock for all LLRFs systems, then wait
         for llrf in self._llrfs:
-            llrf.interlock_manual = 1
+            llrf['IntlkManual-Sel'] = 1
         _time.sleep(1)
         for llrf in self._llrfs:
-            llrf.interlock_manual = 0
+            llrf['IntlkManual-Sel'] = 0
```
Not using the property

anacso17 replied:
done

ericonr commented on siriuspy/siriuspy/orbintlk/main.py:
```diff
@@ -1249,8 +1260,8 @@ def _check_configs(self):
-                fim_orbit = dev.fast_interlock_monitor_orbit
-                fim_manual = dev.fast_interlock_monitor_manual
+                fim_orbit = dev['FIMLLRF1-Sts']
+                fim_manual = dev['FIMManual-Sts']
```
Shouldn't we keep using the property for manual, since that's not going to change depending on the LLRF? It's not a physical connection, after all.

anacso17 replied:
Ok, done

anacso17 commented:
@fernandohds564 , @ericonr , I changed, where was possible, to use properties. I talked to @felipekojihoshino and it seems to be definitive for both LLRF the FIMLLRF1 as orbit interlock input. 

(I have changed to use only PV names directly also because it was annoying me to have two different ways to access the same PV (using the PV name when I need the PV object and using the property when I needed only reading or writing the value), but I see it is interesting to use properties when it is possible, to avoid hard-coded strings. Unfortunately it is not possible to avoid this when we need the PV object itself)

ericonr commented:
Nit: `orbintlk.FIX: fix timing redundancy check` commit body could be wrapped to 72 columns

And I was wondering if maybe the Fout checking shouldn't (eventually) happen in the high level trigger?

anacso17 commented:
> Nit: orbintlk.FIX: fix timing redundancy check commit body could be wrapped to 72 columns

done!
